core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -74,7 +73,6 @@ object UserService {
 class UserService(protected val userInfo: UserInfo,
                   val dataSource: SlickDataSource,
                   protected val gcsDAO: GoogleServicesDAO,
-                  notificationDAO: NotificationDAO,
```

Comment:
NotificationDAO used to be used in UserService back when Rawls handled user registrations, but it's not needed here anymore